addon.xml

```diff
@@ -10,10 +10,12 @@
     <extension point="xbmc.service" library="main.py"/>
     <extension point="xbmc.addon.metadata">
         <summary lang="en_GB">Cast videos from the Youtube application to Kodi</summary>
-        <summary lang="pt_PT">Cast de vídeos a partir da aplicação móvel Youtube para o Kodi</summary>
```

Comment:
@roliverosc please fix the indentation of the file

Reply:
I did not the last time. But now it is fixed. Not?

Reply:
doesn't seem to be :P

Reply:
Fixed? https://github.com/roliverosc/script.tubecast/commit/0192e3918b5911d86afa38d4c637b004daee1af2